tests/core/test_log.py

```diff
@@ -78,13 +99,32 @@ def test_invalid_keywords_raise_exception_on_log_instantiation(ping_setup, kwarg
 def test_log_not_present(ping_setup, w3):
     ping, receipt = ping_setup
     # asserts that every args is not in event log values
-    assert Log(ping.events.Ping, b"y").not_present(receipt)
-    assert Log(ping.events.Ping, first=b"y").not_present(receipt)
-    assert Log(ping.events.Ping, first=b"1").not_present(receipt) is False
-    assert Log(ping.events.Ping, b"y", b"1").not_present(receipt) is False
-    assert Log(ping.events.Ping, b"1", b"2").not_present(receipt) is False
-    assert Log(ping.events.Ping, b"y", second=b"1").not_present(receipt) is False
-    assert Log(ping.events.Ping, first=b"y", second=b"1").not_present(receipt) is False
+    assert Log(ping.events.Ping, b"y".ljust(32, b"\00")).not_present(receipt)
```

Comment:
These assertions look like they each test something different which suggests this test should be parametrized.